src/controller/CommissioningDelegate.h

```diff
@@ -104,6 +104,12 @@ struct WiFiCredentials
     WiFiCredentials(ByteSpan newSsid, ByteSpan newCreds) : ssid(newSsid), credentials(newCreds) {}
 };
 
+struct TermsAndConditionsAcknowledgement
```

Comment:
If this is just focusing on the server end, these bits should not be here either.

Reply:
Removed.